iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -89,7 +92,9 @@ inline optional<T>::optional(in_place_t, Targs&&... args) noexcept
 {
     construct_value(std::forward<Targs>(args)...);
 }
+// AXIVION ENABLE STYLE AutosarC++19_03-A12.6.1
 
+// AXIVION Next Construct AutosarC++19_03-A13.3.1 : It's an abstract call and cause no confusion
```

Comment:
I think it's a false positive, please see [this comment](https://github.com/eclipse-iceoryx/iceoryx/pull/1607#discussion_r984384205). @elfenpiff What do you think? 
If this is true, the justification could be something like "False positive. Overloading excluded via std::enable_if in typename std::enable_if<!std::is_same<U, optional<T>&>::value, optional>::type& operator=(U&& value) noexcept"

Reply:
@FerdinandSpitzschnueffler updated the justification as per @elfenpiff . thanks

Reply:
@saif-at-github @FerdinandSpitzschnueffler maybe I put this justification in the wrong thread or answered to the wrong question but the justification was meant for instance for the functional interface class.

Reply:
@saif-at-github Can you please update the justification? Maybe you can reuse my first comment on this.

Reply:
@FerdinandSpitzschnueffler  this is addressed. Thanks